src/pages/workspace/accounting/intacct/import/SageIntacctEditUserDimensionsPage.tsx

```diff
@@ -85,8 +86,9 @@ function SageIntacctEditUserDimensionsPage({route}: SageIntacctEditUserDimension
                 shouldValidateOnChange
             >
                 <OfflineWithFeedback
-                    pendingAction={config?.pendingFields?.[`dimension_${editedUserDimensionName}`]}
-                    errors={config?.errorFields?.[`dimension_${editedUserDimensionName}`]}
+                    pendingAction={settingsPendingAction([`dimension_${editedUserDimensionName}`], config?.pendingFields)}
```

Comment:
Just a small code clean-up: can we change that "dimension_" string to CONST? Similarly to `CONST.XERO_CONFIG.TRACKING_CATEGORY_PREFIX`. I should have done this when I was implementing SageIntacctEditUserDimensionsPage, but I missed it 😅